.github/workflows/format.yml

```diff
@@ -0,0 +1,47 @@
+name: Format
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  edit:
+    name: clang-format
+    if: ${{ github.event.comment.body == '@crtrott format' }}
+    runs-on: ubuntu-latest
+    env:
+      CLANG_FORMAT: clang-format-8
+    steps:
+      - name: Check if message comes from a PR
+        uses: octokit/request-action@v2.x
+        id: issue_info
+        with:
+          route: GET /repos/:repository/issues/:pull_number
+          repository: ${{ github.repository }}
+          pull_number: ${{ github.event.issue.number }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Who provides the credentials?

Reply:
@dalg24, I think this is like a macro for github to put your token in the message they send the computer running the action.  It's very possible that I am wrong, but I think you make up your own secret.

Reply:
See https://docs.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token and https://docs.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token#permissions-for-the-github_token.